scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -1333,8 +1379,9 @@ def _benchmark_cnn(self):
       log_fn('Running warm up')
       local_step = -1 * self.num_warmup_batches
 
-      if self.single_session or (self.params.cross_replica_sync and
-                                 self.params.job_name):
+      if (self.single_session or (self.params.cross_replica_sync and
+                                 self.params.job_name) or
```

Comment:
Add another space before self.params.job_name to line it up with the parenthesis